src/actions/arcActions.ts

```diff
@@ -163,24 +159,17 @@ export function redeemReputationFromToken(scheme: Scheme, addressToRedeem: strin
       };
       const gasEstimate = await arc.web3.eth.estimateGas(txToSign);
       txToSign.gas = gasEstimate;
-      console.log(`estimated gas cost           : ${gasEstimate  * gasPrice}`);
-      // console.log(txToSign);
       // if the gas cost is higher then the users balance, we lower it to fit
       const userBalance = await arc.web3.eth.getBalance(redeemerAddress);
-      console.log(`balance of ${redeemerAddress}: ${userBalance}`);
       if (userBalance < gasEstimate * gasPrice) {
         txToSign.gasPrice = Math.floor(userBalance/gasEstimate);
       }
-      console.log(`estimated gas cost after adjusting: ${gasEstimate  * txToSign.gasPrice}`);
-      const signedTransaction = await arc.web3.eth.accounts.signTransaction(txToSign, privateKey);
-      // console.log(signedTransaction);
+      // const signedTransaction = await arc.web3.eth.accounts.signTransaction(txToSign, privateKey);
       dispatch(showNotification(NotificationStatus.Success, "Sending redeem transaction, please wait for it to be mined"));
-      const txHash = await arc.web3.utils.sha3(signedTransaction.rawTransaction);
-      console.log(`transaction hash: ${txHash}`);
+      // const txHash = await arc.web3.utils.sha3(signedTransaction.rawTransaction);
```

Comment:
are we not using this code?

Reply:
Same deal:  I removed the console.log which was the only place it was being used.  It is not my code and I didn't want to delete someone else's potentially useful/informative code.